packages/api/src/lib/structures/http/Server.ts

```diff
@@ -1,10 +1,27 @@
 import type { SapphireClient } from '@sapphire/framework';
+import { EventEmitter } from 'events';
 import { createServer as httpCreateServer, Server as HttpServer } from 'http';
```

Comment:
Wouldn't it be better to import from `http2`?
```ts
import { createServer as http2CreateServer, Http2Server } from 'http2';
```

Reply:
Http2 requires SSL, and so does Https.